packages/sdk-codegen/src/python.gen.ts

```diff
@@ -105,6 +105,7 @@ export class PythonGen extends CodeGen {
     string: { default: this.nullStr, name: 'str' },
     uri: { default: this.nullStr, name: 'str' },
     void: { default: this.nullStr, name: 'None' },
+    any: { default: this.nullStr, name: 'Any' },
```

Comment:
The import statement in `methodsPrologue` and `modelsPrologue` need to be modified to include `Any` from the `typing` module. Right now it's not recognized causing `mypy` to complain.

Once that's done, `mypy` will still throw some 805 errors because of any generated model with at least one attribute annotated with `Any`. We might need to remove https://github.com/looker-open-source/sdk-codegen/blob/875d5cd9a0213abb5b257328b88602c43c618c36/python/mypy.ini#L4 as I don't see a way to dial this down a notch from [here](https://mypy.readthedocs.io/en/stable/command_line.html).

Reply:
Thanks for confirming an import statement is required. I hadn't yet researched which import was appropriate because I was trying to wrap up for the night.

This will require another round of change/publish/upgrade/test but I'll wait until Monday for it.